Libraries/CMSIS/Device/Maxim/MAX32675/Source/system_max32675.c

```diff
@@ -153,9 +153,11 @@ __weak void SystemInit(void)
     __DSB();
     __ISB();
 
-    /* Change system clock source to the main high-speed clock */
-    MXC_SYS_Clock_Select(MXC_SYS_CLOCK_IPO);
-    SystemCoreClockUpdate();
+    /* The default clock for the MAX32675 Revision B is the IBRO. Do not update. */
+    if ((MXC_GCR->revision & MXC_F_GCR_REVISION_REVISION) == 0x03B4) {
```

Comment:
Should the reverse logic be used?  That is, only do the clock change if rev is not Rev B.

Reply:
Removed the entire if statement.